### DISCUSSION TITLE
Server crashing

### DISCUSSION THREAD

**Discussion by mkjh01** [jamulussoftware/jamulus#1335] [Technical Support & Help]
I have set up a Jamulus server on Amazon lightsail for our orchestra .  It keeps crashing after about an hour and I think this is because  record is on. Can I turn this off? OR how do I go back into the program to take the relevant line of code out? (I am a programming novice!)
Thank you

**Comment by jujudusud** [comment]
Hi,
You have to set the right option when starting your server.
To do so, you should see this config web page : https://jamulus.io/wiki/Command-Line-Options

**Reply by mkjh01** [reply]
How do I do this if the server is in the web?

**Reply by jujudusud** [reply]
I have not vert much knowledge in Amazon lightsail serveurs but, I have some questions :
- which OS did you choose ? Windows , Ubuntu ? Another one ?
- how did you install Jamulus on it ?
- how did you start the server ?

The options are communicanted to the server when it is started with an icon if it is the GUI version or by a command line or a script file if it is a headless version.

**Comment by hoffie** [comment]
What version are you running? 3.7.0 fixes memory leaks in the recording code, so if you haven't updated yet, you should. :)

I think there are still other unresolved issues -- Jamulus may exit if recording is active and you run out of disk space, for example. This is worth checking as well.

**Reply by mkjh01** [reply]
Hi, I have updated my version of Jamulus but I think I need to change a command line on the server, but I have no idea how to access it on the web?

**Comment by mkjh01** [comment]
Hi,
I think it is UbuntuI installed by opening an Instance on Amazon Lightsail and copied and pasted the command lines I was givenI literally pressed set up and away it went!!
I can get into my Instance but I can't see any of the command lines
Giving a gal a little knowledge, or not in my case, is a dangerous thing.


-----Original Message-----

**Reply by jujudusud** [reply]
https://www.google.com/url?sa=t&source=web&rct=j&url=https://m.facebook.com/notes/jamulus-worldjam/howto-idiots-guide-to-installing-or-upgrading-a-jamulus-server-on-amazon-aws-lig/818091045662521/&ved=2ahUKEwiN_bGZsMvvAhVx5-AKHX1yDZYQFjAAegQIAxAC&usg=AOvVaw2mEBTGQo9ig7WjccN1mHAm

There is some informations here and I'm sure  Simon Tomlinson will be able to help more than me.

The principle is to adapt the lunch script with the right option over ssh connections on the server.

**Comment by mkjh01** [comment]
Thank you for your help. I'll give it a go
On Thursday, 25 March 2021 Julien Taverna ***@***.***> wrote:



https://www.google.com/url?sa=t&source=web&rct=j&url=https://m.facebook.com/notes/jamulus-worldjam/howto-idiots-guide-to-installing-or-upgrading-a-jamulus-server-on-amazon-aws-lig/818091045662521/&ved=2ahUKEwiN_bGZsMvvAhVx5-AKHX1yDZYQFjAAegQIAxAC&usg=AOvVaw2mEBTGQo9ig7WjccN1mHAm

There is some informations here and I'm sure Simon Tomlinson will be able to help more than me.

The principle is to adapt the lunch script with the right option over ssh connections on the server.
